packs/data/pathfinder-bestiary-3.db/android-infiltrator.json

```diff
@@ -671,7 +671,42 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "nanite-surge",
+                        "toggleable": true,
+                        "value": false
+                    },
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "nanite-surge"
+                        ],
+                        "selector": "skill-check",
+                        "type": "status",
+                        "value": 2
+                    },
+                    {
+                        "key": "TokenLight",
+                        "predicate": [
+                            "nanite-surge"
+                        ],
+                        "type": "torch",
+                        "value": {
+                            "animation": {
+                                "intensity": "4",
+                                "speed": "1",
+                                "type": "torch"
+                            },
+                            "bright": "0",
+                            "color": "#9b7337",
+                            "dim": "10",
+                            "shadows": "0.2"
+                        }
+                    }
```

Comment:
```suggestion
                    {
                        "key": "TokenLight",
                        "value": {
                            "bright": "0",
                            "color": "#3b7355",
                            "dim": "10",
                            "shadows": "0.1",
                            "luminosity": 0.4
                        }
                    }
```
The torch animation looks weird in this case and I'd associate a more muted blue with a dim glow from an android personally. Here's what it looks like in darkness and dim light.

![image](https://user-images.githubusercontent.com/80183198/199851230-10ecd3b0-2d14-4e62-bffa-937c597820f1.png)

![image](https://user-images.githubusercontent.com/80183198/199851269-0c04663f-f9fb-4c83-8902-af716dd4c3f9.png)